i18n/qfield_en.ts

```diff
@@ -1583,6 +1583,10 @@ The features geometries will be combined into feature &apos;%1&apos;, which will
         <source>Duplicate Feature</source>
         <translation>Duplicate Feature</translation>
     </message>
+    <message>
```

Comment:
Can you remove the translation modifications here? Let transifex deal with it automatically.

Reply:
Should I remove all translations modification that i made (fr,es,it) ?

Reply:
@qsavoye , yes, please, all translation modifications should be reverted.